homeassistant/components/hue/config_flow.py

```diff
@@ -230,6 +231,13 @@ async def async_step_ssdp(self, discovery_info: ssdp.SsdpServiceInfo) -> FlowRes
         if not url.hostname:
             return self.async_abort(reason="not_hue_bridge")
 
+        # Ignore if host is IPv6
```

Comment:
If you wanted to avoid some lines, we have `from homeassistant.util.network import is_ipv6_address`